cmd/csv.go

```diff
@@ -40,7 +41,11 @@ func initCSV() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer f.Close()
+	defer func() {
```

Comment:
probably better to write another function and call it then creating more inline functions? it would also make the code more readable